apps/meteor/app/models/server/models/LivechatVisitors.js

```diff
@@ -200,7 +202,7 @@ export class LivechatVisitors extends Base {
 
 	findOneGuestByEmailAddress(emailAddress) {
 		const query = {
-			'visitorEmails.address': new RegExp(`^${escapeRegExp(emailAddress)}$`, 'i'),
+			'visitorEmails.address': emailAddress,
```

Comment:
I know I proposed these changes in the first place, but taking a wider look now, I wonder if we should do `String(emailAddress).toLowerCase()` here, instead of changing the case in every place using this function.. wdyt?